spec/support/factories/claimant_factory.rb

```diff
@@ -55,4 +55,30 @@
     employment_end {"0/0/0"}
     disagree_employment {nil}
   end
+
+  trait :claimant_invalid_dates do
+    claimants_name { Faker::Name.name }
```

Comment:
It looks like there are alot of values in here that are nothing to do with dates.  Did you know that you can call one trait from within another - so if the other values that you wanted are already in another trait and you just want to override the dates - just put the trait name in - its just defined a method so if the trait was called 'fred' then

trait :claimant_invalid_dates do
  fred
  date_field_i_want_to_change { value }